### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by Lordisch**

### CODE REVIEW

File: src/components/SettingComponents/Faq.vue [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<template>
+  <div>
+    <SettingTitle docPath="/guide/faq">
+      {{ $t('_faq.title') }}
+    </SettingTitle>
+    <v-row no-gutters> <!-- das brauchen wir nicht -->
+    <v-col>  <!-- das brauchen wir nicht -->
+    <v-list>
```
Comment (rFranzr) [inline src/components/SettingComponents/Faq.vue]: Remove comment from HTML

File: src/components/SettingComponents/Faq.vue [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                  </v-icon>
+                </v-btn>
+              </v-col>
+            </v-row>
+          </v-list-item>
+        </div>
+      </draggable>
+    </v-list>
+    </v-col>  <!-- das brauchen wir nicht -->
+    </v-row>  <!-- das brauchen wir nicht -->
```
Comment (rFranzr) [inline src/components/SettingComponents/Faq.vue]: Remove comments

File: src/components/SettingComponents/Faq.vue [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    <v-divider />
+    <v-col cols="12" md="6">
```
Comment (rFranzr) [inline src/components/SettingComponents/Faq.vue]: This col is probably unnecessary

File: src/components/SettingComponents/Faq.vue [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+            {{ $t('_faq.reset') }}
+          </span>
+      </v-tooltip>
+    </v-col>
+    <DialogForm :title="$t('_faq.addQuestionDialog')" icon="mdi-chat-question"
+                ref="addQuestionDialog"
+                :form-schema="faqAddSchema"
+                @submit="addQuestion">
+      <template v-slot:title-after>
+        <Editor v-model="content"/> <!-- Editor für das hinzufügen einer neuen Antwort. -->
```
Comment (rFranzr) [inline src/components/SettingComponents/Faq.vue]: Remove comment

File: src/components/SettingComponents/Faq.vue [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+      </template>
+    </DialogForm>
+    <DeleteConfirmationDialog ref="deleteQuestionConfirmationDialog"
+                              @submit="deleteQuestion"/>
+    <DialogForm :title="$t('_faq.editQuestionDialog')" icon="mdi-chat-question"
+                ref="editQuestionDialog"
+                :form-schema="faqAddSchema"
+                @submit="editQuestion">
+      <template v-slot:title-after>
+        <Editor v-model="content"/> <!-- Editor für das Editieren einer bestehenden Antwort. -->
```
Comment (rFranzr) [inline src/components/SettingComponents/Faq.vue]: Comment

File: src/components/SettingComponents/Faq.vue [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+  components: {
+    DialogForm,
+    SettingTitle,
+    DeleteConfirmationDialog,
+    Editor,
+  },
+  data() {
+    return {
+      questions: null,
+      content: null, // content des Editors
```
Comment (rFranzr) [inline src/components/SettingComponents/Faq.vue]: Comment

File: src/components/SettingComponents/Faq.vue [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+      (await openapi).faq_editQuestion(question.id, data).then(() => {
+        this.fetchData();
+        this.$notify({
+          title: this.$t('_faq.messages.editSuccess'),
+          type: 'success',
+        });
+        this.content = null;
+        this.$refs.editQuestionDialog.closeAndReset();
+      }).catch((err) => {
+        this.$refs.editQuestionDialog.setErrorMessage(err.response.data.details);
```
Comment (rFranzr) [inline src/components/SettingComponents/Faq.vue]: err.response.data.detail

File: src/forms/FaqForm.ts [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import i18n from '@/plugins/i18n';
+
+export default {
+  type: 'object',
+  required: [
+    'title', 'content',
```
Comment (rFranzr) [inline src/forms/FaqForm.ts]: Content not required as no existing anymore

File: src/views/Settings.vue [hunk 9]
```diff
@@ -116,6 +116,12 @@ export default {
           reqProp: 'server_show',
           title: this.$t('servers'),
         },
+        {
+          name: 'faq',
+          icon: 'mdi-chat-question',
+          component: 'Faq',
```
Comment (ibot3) [inline src/views/Settings.vue]: Rename.

File: src/views/Faq.vue [hunk 10]
```diff
@@ -0,0 +1,42 @@
+<template>
```
Comment (ibot3) [inline src/views/Faq.vue]: Rename this file form `Faq` to `FAQ`.

File: src/router/index.ts [hunk 11]
```diff
@@ -141,6 +141,12 @@ const routes: Array<RouteConfig> = [
     meta: { title: i18n.t('_pageTitle.team') },
     component: () => import('../views/Team.vue'),
   },
+  {
+    path: '/faq',
+    name: 'FAQ',
+    meta: { title: i18n.t('_pageTitle.faq') },
+    component: () => import('../views/Faq.vue'),
```
Comment (ibot3) [inline src/router/index.ts]: Rename.

**Review by rFranzr — APPROVED** [review]

**Review by ibot3 — APPROVED** [review]

**Pull request closed without merging**